shared-doc/build-and-deploy-the-quickstart.adoc

```diff
@@ -94,3 +94,21 @@ $ mvn wildfly:deploy
 This deploys the `{archiveDir}/{archiveFileName}` to the running instance of the server.
 
 You should see a message in the server log indicating that the archive deployed successfully.
+
+== Access the quickstart application
+
+You can either access the application via your browser at http://localhost:8080/microprofile-lra/. You can also access it from the command line:
+
+[source,bash]
+----
+$ curl http://localhost:8080/microprofile-lra/
+----
+
+It should return a simple output:
+
+[source]
+-----
+Hello! Microprofile-Lra Quickstart deployed successfully! You can find the available operations in the included README file.
```

Comment:
Same here

Reply:
please revert this change, it would add the new content to all quickstart READMEs

Reply:
Reverted.